src/quo/components/links/internal_link_card/channel/style.cljs

```diff
@@ -0,0 +1,42 @@
+(ns quo.components.links.internal-link-card.channel.style
+  (:require [quo.foundations.colors :as colors]))
+
```

Comment:
why are we using so many fixed sidths in this file? 🤔 - I think a lot of them should be using margins/ padding instead

Reply:
what are the other loadings doing? I would think we ought to achieve this with margin. Can it be done? 
cc @FFFra